yt_dlp/extractor/odkmedia.py

```diff
@@ -0,0 +1,139 @@
+import json
+import urllib.error
+
+from .common import InfoExtractor
+from ..utils import (
+    ExtractorError,
+    GeoRestrictedError,
+    float_or_none,
+    traverse_obj,
+    try_call
+)
+
+
+class OnDemandChinaEpisodeIE(InfoExtractor):
+    _VALID_URL = r'https?://www\.ondemandchina\.com/\w+/watch/(?P<series_name>[\w-]+)/(?P<id>ep-(?P<episode_num>\d+))'
+    _TESTS = [{
+        'url': 'https://www.ondemandchina.com/en/watch/together-against-covid-19/ep-1',
+        'info_dict': {
+            'id': '264394',
+            'ext': 'mp4',
+            'duration': 3256.88,
+            'title': 'EP 1 The Calling',
+            'alt_title': '第1集 令出如山',
+            'thumbnail': 'https://d2y2efdi5wgkcl.cloudfront.net/fit-in/256x256/media-io/2020/9/11/image.d9816e81.jpg',
+            'description': '疫情严峻，党政军民学、东西南北中协同应考',
+            'tags': ['Social Humanities', 'Documentary', 'Medical', 'Social'],
+        }
+    }]
+
+    _QUERY = """query Episode($programSlug: String!, $episodeNumber: Int!, $kind: String!, $part: Int) {
+        episode(
+            programSlug: $programSlug
+            episodeNumber: $episodeNumber
+            kind: $kind
+            part: $part
+        ) {
+            ...EpisodeDetail
+            __typename
+            }
+        }
+
+        fragment EpisodeDetail on Episode {
+            id
+            episodeNumber
+            part
+            kind
+            link
+            title
+            titleEn
+            titleKo
+            titleZhHans
+            titleZhHant
+            synopsis
+            synopsisEn
+            synopsisKo
+            synopsisZhHans
+            synopsisZhHant
+            seoSynopsis
+            seoKeyword
+            videoCcLanguages
+            videoDuration
+            releaseDate
+            people {
+                ...PeopleItem
+                __typename
+            }
+            images {
+                thumbnail
+                __typename
+            }
+            ageRating {
+                isBlock
+                __typename
+            }
+            __typename
+            }
+            fragment PeopleItem on People {
+                role
+                slug
+                nameEn
+                nameZhHans
+                nameZhHant
+                __typename
+            }
+            """
```

Comment:
There's so much metadata here that you aren't using. If you can actually extract them, that would be best. If not, remove them from query

Reply:
I've removed them for now. Let me know if you want to extract any more fields

Reply:
@HobbyistDev